package.json

```diff
@@ -18,19 +19,20 @@
         "prepare": "npm run build"
     },
     "dependencies": {
-        "@ethereumjs/block": "4.3.0",
-        "@ethereumjs/common": "3.2.0",
-        "@ethereumjs/evm": "1.4.0",
-        "@ethereumjs/statemanager": "1.1.0",
-        "@ethereumjs/tx": "4.2.0",
-        "@ethereumjs/vm": "6.5.0",
+        "@ethereumjs/block": "5.2.0",
+        "@ethereumjs/common": "4.3.0",
+        "@ethereumjs/evm": "3.0.0",
+        "@ethereumjs/rlp": "5.0.2",
+        "@ethereumjs/statemanager": "2.3.0",
+        "@ethereumjs/tx": "5.3.0",
+        "@ethereumjs/util": "9.0.3",
+        "@ethereumjs/vm": "8.0.0",
         "cbor": "^9.0.2",
-        "ethereum-cryptography": "2.1.3",
-        "ethereumjs-util": "^7.1.5",
+        "ethereum-cryptography": "2.2.1",
         "fs-extra": "^11.2.0",
```

Comment:
Can move this to the dev dependencies

Reply:
[a79cec9](https://github.com/Consensys/sol-dbg/pull/41/commits/a79cec9eee0cdf560102b9d5cd98f305257c2250)